docs/index.md

```diff
@@ -70,7 +70,7 @@ Installation instructions are available [here](./ICMClusterInstallation).
 
 ### Dataset converters (`clinica convert`)
 
-Clinica provides tools to curate several publicly available neuroimaging datasets and convert them to BIDS namely:
+Clinica provides tools to curate several publicly available neuroimaging datasets and convert them to [BIDS](BIDS) namely:
```

Comment:
This reference feels better than the others. Maybe we can forward all the BIDS reference in our documentation to the `BIDS.md` and link the BIDS website only there ? It would avoid future dead BIDS website links, and a handful of potential updates to make.